internal/armada/configuration/types.go

```diff
@@ -123,6 +123,11 @@ type PreemptionConfig struct {
 	// Priority class assigned to pods that do not specify one.
 	// Must be an entry in PriorityClasses above.
 	DefaultPriorityClass string
+
+	// TerminationGracePeriod settings for preemptible pods
```

Comment:
I disagree with having termination period be related to preemption. I saw Chris suggested moving it here, but I think these settings should be in ArmadaConfig and that they should be applied to all jobs, regardless of if they have a priority class etc.